frontend/ui/elements/common_info_menu_table.lua

```diff
@@ -1,6 +1,8 @@
 local Device = require("device")
 local InfoMessage = require("ui/widget/infomessage")
 local UIManager = require("ui/uimanager")
+local Event = require("ui/event")
```

Comment:
Event should go a little higher, see https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires